src/formatter.coffee

```diff
@@ -0,0 +1,109 @@
+MESSAGE_RESERVED_KEYWORDS = ['channel','group','everyone','here']
+
+
+# https://api.slack.com/docs/formatting
+class SlackFormatter
+
+  constructor: (@dataStore) ->
+
+
+  ###
+  Formats links and ids
+  ###
+  links: (text) ->    
```

Comment:
@DEGoodmanWilson I think we can replace this function with [`{parse: true}`](https://api.slack.com/methods/chat.postMessage#formatting)